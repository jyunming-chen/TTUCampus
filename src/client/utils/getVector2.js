// @ts-check

import { Vector2 } from 'three';

/**
 * @param {number[]} xyArray
 */
export default function getVector2(xyArray) {
  const [x, y] = xyArray;
  return new Vector2(x, y);
}
