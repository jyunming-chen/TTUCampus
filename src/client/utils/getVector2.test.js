import getVector2 from './getVector2';

test('getVector2([1, 2])', () => {
  expect(getVector2([1, 2])).toMatchSnapshot();
});
