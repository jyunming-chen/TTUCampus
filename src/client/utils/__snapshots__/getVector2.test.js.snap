// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getVector2([1, 2]) 1`] = `
Vector2 {
  "x": 1,
  "y": 2,
}
`;
